.gitignore

```diff
@@ -16,3 +16,4 @@ tox.ini
 test-reports/*
 config.json
 .eggs/
+dynamodb-data/
```

Comment:
Should we update the readme to tell the user an alternative way to run local dynamo is to use the docker-compose.yml file ?